src/main/java/org/cryptomator/ui/keyloading/hub/RegisterDeviceController.java

```diff
@@ -95,35 +104,61 @@ private String determineHostname() {
 
 	@FXML
 	public void register() {
-		deviceNameAlreadyExists.set(false);
-		registerBtn.setContentDisplay(ContentDisplay.LEFT);
-		registerBtn.setDisable(true);
-
-		var keyUri = URI.create(hubConfig.devicesResourceUrl + deviceId);
-		var deviceKey = keyPair.getPublic().getEncoded();
-		var dto = new CreateDeviceDto(deviceId, deviceNameField.getText(), BaseEncoding.base64Url().omitPadding().encode(deviceKey));
-		var json = toJson(dto);
-		var request = HttpRequest.newBuilder(keyUri) //
+		workInProgress.set(true);
+
+		var apiRootUrl = hubConfig.getApiBaseUrl();
+		var deviceKey = deviceKeyPair.getPublic().getEncoded();
```

Comment:
Remove variable and inline the function call where needed. Otherwise it is hard to understand, why the "devicekey" is not used in the userkey encryption. (because this is the encoded form)